src/py/flwr/server/app.py

```diff
@@ -265,10 +265,14 @@ def run_superlink() -> None:
     # Obtain certificates
     certificates = try_obtain_server_certificates(args, args.fleet_api_type)
 
+    # Disable the user auth TLS check if args.disable_oidc_tls_cert_verification is
+    # provided
+    user_auth_tls_check = getattr(args, "disable_oidc_tls_cert_verification", True)
```

Comment:
Shouldn't the default here be `False`?

Reply:
Also, should we rename the variable a bit for consistency?
```suggestion
    disable_oidc_tls_cert_verification = getattr(args, "disable_oidc_tls_cert_verification", True)
```